src/CopyImportRedshift.php

```diff
@@ -29,7 +29,7 @@ protected function importDataToStagingTable($stagingTempTableName, $columns, $so
             }, $columns)
         ) . ") ";
 
-        $sql .= "SELECT " . implode(',', array_map(function ($column) use ($sourceColumnTypes) {
+        $sql .= "SELECT " . implode(',', array_map(function ($column) use ($sourceColumnTypes, $options) {
```

Comment:
tvl tohle je zajimavy. To znamena ze ta podminka co tam je v tom tedle te closure dale nikdy nefungovala? :)

Reply:
jako ze `isset($options["convertEmptyValuesToNull"]) ` bylo vzdy `false`?

Reply:
Presne tak, vzdycky to bylo `false` a ta podminka nikdy nefungovala.